libs/wire-api/src/Wire/API/Routes/Public.hs

```diff
@@ -275,3 +286,74 @@ instance HasServer api ctx => HasServer (OmitDocs :> api) ctx where
 
 instance RoutesToPaths api => RoutesToPaths (OmitDocs :> api) where
   getRoutes = getRoutes @api
+
+--------------------------------------------------------------------------------
+-- Z-OAuth
+
+data ZUserOrOAuth (scope :: OAuthScope)
```

Comment:
```suggestion
-- FUTUREWORK: it would be nice to have unit tests for this and the instances (esp. `HasServer`, but we cover this with integration tests in brig et al. for now.
data ZUserOrOAuth (scope :: OAuthScope)
```

(right?)

Reply:
That's right